src/ldclient_context_filter.erl

```diff
@@ -8,7 +8,8 @@
 
 %% API
 -export([
-    format_context_for_event/2
+    format_context_for_event/2,
```

Comment:
Do we have a use-case for the version without anonymous redaction?

Reply:
There isn't an explicit use case called out in the internal discussion, but the change does stipulate that we should only redact those attributes for feature events, not every event. Is that what you mean?

Reply:
Ok. So we redact all attributes for anonymous contexts only in the feature event, but would keep them in an identify.